peerscout/keyword_extract/keyword_extract.py

```diff
@@ -144,26 +146,28 @@ def etl_keywords_get_latest_state(
             keyword_extract_config.text_field
         ]
     )
-    write_to_file(processed_data, full_file_location)
-    create_or_extend_table_schema(
-        keyword_extract_config.gcp_project,
-        keyword_extract_config.destination_dataset,
-        keyword_extract_config.destination_table,
-        full_file_location
-    )
-    write_disposition = (
-        WriteDisposition.WRITE_APPEND
-        if keyword_extract_config.table_write_append
-        else WriteDisposition.WRITE_TRUNCATE
-    )
-    load_file_into_bq(
-        filename=full_file_location,
-        table_name=keyword_extract_config.destination_table,
-        auto_detect_schema=True,
-        dataset_name=keyword_extract_config.destination_dataset,
-        write_mode=write_disposition,
-        project_name=keyword_extract_config.gcp_project
-    )
+
+    with NamedTemporaryFile() as named_temp_file:
+        write_to_file(processed_data, named_temp_file.name)
```

Comment:
It might still be worth naming the variable to what it represents. e.g. `temp_processed_jsonl_path`.